.github/actions/configure_4C/action.yml

```diff
@@ -32,7 +32,7 @@ runs:
         cd $GITHUB_WORKSPACE
         ./utilities/set_up_dev_env.sh
         cd $BUILD_DIRECTORY
-        cmake $GITHUB_WORKSPACE --fresh --preset=$CMAKE_PRESET $ADDITIONAL_CMAKE_FLAGS | tee $LOG_FILE
+        echo $GITHUB_WORKSPACE --fresh --preset=$CMAKE_PRESET $ADDITIONAL_CMAKE_FLAGS | xargs cmake | tee $LOG_FILE
```

Comment:
What is the reason for this special invocation?

Reply:
We need this so that `$ADDITIONAL_CMAKE_FLAGS` can contain multiple flags (and not just one).

Reply:
I would add a comment explaining this reason, otherwise somebody is going to "optimize" the call.